docs/api/desktop-capturer.md

```diff
@@ -62,7 +62,7 @@ stopButton.addEventListener('click', () => {
   <body>
     <button id="startButton" class="button">Start</button>
     <button id="stopButton" class="button">Stop</button>
-    <video width="320" height="240" autoplay></video>
+    <video width="320" height="240" autoplay controls></video>
```

Comment:
DX improvement: `controls` being visible is important for the dev to be able to easily determine if there's an audio track available or not.